docs/sources/setup-grafana/configure-security/configure-team-sync.md

```diff
@@ -16,7 +16,7 @@ weight: 1000
 
 Team sync lets you set up synchronization between your auth providers teams and teams in Grafana. This enables LDAP, OAuth, or SAML users who are members of certain teams or groups to automatically be added or removed as members of certain teams in Grafana.
 
-> **Note:** Available in [Grafana Enterprise](../../../introduction/grafana-enterprise/) and [Grafana Cloud Advanced](/docs/grafana-cloud/).
+> **Note:** Available in [Grafana Enterprise](../../../introduction/grafana-enterprise/) and [Grafana Cloud Pro and Advanced](/docs/grafana-cloud/).
```

Comment:
@vtorosyan I was wondering if there's any reason this link doesn't use `<GRAFANA_VERSION>` like the rest of the links?

Reply:
Cloud docs are not coupled with Grafana versions, so we just straight redirect to the main page https://grafana.com/docs/grafana-cloud/